src/components/GeoMap.js

```diff
@@ -423,9 +423,9 @@ export default class GeoMap {
 
       dataset.forEach(d => {
         d.rollupValue[attrKey] = attrValue.rollup(d.values);
-      });
+    });
 
-      const scaleDomain = d3.extent(dataset, d => d.rollupValue[attrKey]);
+      const scaleDomain = d3.extent(dataset, d => Number(d.rollupValue[attrKey]));
```

Comment:
this, or the other call to ``Number()`` has made the column do an ASCII instead of a natural sort.  The net result is that it orders the ``widget`` column in the ``negative.example.html`` example as "-10, -100 -110, -40..." :(

while I know in the ``columnsDetails`` you can define a  ``dataParse`` function, i was just surprised that it defaulted to ASCII.  We  should do regression checks to ensure this doesn't change the existing default sort behavior.